README.md

```diff
@@ -4,24 +4,62 @@
 Experimental webview editor for [nih-plug](https://github.com/robbert-vdh/nih-plug) using [wry](https://github.com/tauri-apps/wry).
 Built on top of [baseview](https://github.com/RustAudio/baseview).
 
-## Current status
+## Compatability
 
-I've only been able to test this on macOS so far on which it has been working quite robustly.
-Now that wry officially supports attaching to an existing window handle, I'm hoping that Windows also works but this needs to be verified.
+Currently, Windows and macOS are supported and have been tested.
+
+## Issues
 
 On macOS there is an unresolved issue where pressing the escape key in Ableton Live will lead to a crash.
 I've reported this to Ableton, and I'm currently mitigating this by consuming the escape keypress behind the scenes.
 
 ## Features
-- send arbitrary JSON values back and forth to the webview using Serde
-- resizable plug-in window
-- drag and drop files with full paths
-- callback for deciding which key events from DAW to consume 
-- customisable background color for when the view is still loading (avoid initial flash of white)
-- use devtools
+- Send arbitrary JSON values back and forth to the webview using Serde
+- Resizable plug-in window
+- Drag and drop files with full paths
+- Callback for deciding which key events from DAW to consume 
+- Customisable background color for when the view is still loading (avoid initial flash of white)
+- Use devtools
 
 ## Usage
 
-[Check out the example.](https://github.com/maxjvh/nih-plug-webview/blob/main/example/src/)
+### Web Framework (Recommended)
```

Comment:
maybe explain before this chapter that there are two ways to set this up, using a web framework or with plain html etc